forte/pymodule.py

```diff
@@ -514,7 +526,8 @@ def run_forte(name, **kwargs):
 
     if (options.get_bool("CASSCF_REFERENCE") or job_type == "CASSCF"):
         if options.get_str('INT_TYPE') == 'FCIDUMP':
-            raise Exception('Forte: the CASSCF code cannot use integrals read' ' from a FCIDUMP file')
```

Comment:
I wonder would it be oneliner if removing `'  '` after read?

Reply:
Fixed.